docs/running-jobs/job-and-queue-scheduling.md

```diff
@@ -1,55 +1,128 @@
 
 ## Introduction
-At a high level, getting computational tasks run on systems at ALCF is a two step process:
+At a high level, getting computational tasks run on an HPC system is a two step process:
 
-1. You request and get allocated resources (compute nodes) on one or more of the systems.  This is accomplished by interacting with the job scheduler / workload manager.  In the ALCF we use PBS Professional.
-2. You execute your tasks on those resources.  This is accomplished in your job script by interacting with various system services (MPI, OpenMP, the HPE PALS job launch system, etc.)
+1. You request and get allocated resources (we allocate at the node level, but some facilities you request number of cores and RAM, etc.) on one or more of the systems.  This is accomplished by interacting with the job scheduler / workload manager.  In the ALCF we use PBS Professional.
+2. You execute your tasks on those resources.  This is accomplished in your job script by interacting with various system services (MPI, OpenMP, the HPE PALS task launch system, etc.)
 
 Our documentation is organized in two sections aligned with the two steps described above.
 
+## Table of Contents
+ - **Obtaining and managing compute resources at ALCF - General PBS information common to all systems**
+     - [Definitions and Notes](#Definitions-and-Notes)
+     - [Quick Start](#Quick-Start)
+	 - [qsub - submit a job to run](#qsub)
+	 - [qstat - query the status of jobs/queues](#qstat)
+	 - [qalter - alter a queued job](#qalter)
+	 - [qdel - delete a queued or running job](#qdel)
+	 - [qmove - move a job to a different queue](#qmove)
+	 - [qhold,qrls - place/release a hold on a job in a queue](#qhold,qrls)
+	 - [qselect - utility to select jobids that meet criteria](#qselect)
+	 - [qmsg - write a message into a jobs output file](#qmsg)
+	 - [qsig - send a signal to a job](#qsig)
+	 - [pbsnodes - Get information about the current state of nodes](#pbsnodes)
+ - **Obtaining and managing compute resources at ALCF - Machine Specific Information**
+     - [Polaris](#Polaris)
+         - [Queues](#Polaris-Queues)
+         - [Network: Rack and Dragonfly Group Mappings](#Rack-and-Dragonfly-Group-Mappings)
+         - [Compute Node Access to the Internet](#Compute-Node-Access-to-the-Internet)
+ - **Controlling task execution on your allocated resources**
+    - [Running MPI+OpenMP Applications](#Running-MPI+OpenMP-Applications)
+    - [Running GPU-enabled Applications](#Running-GPU-enabled-Applications)
+    - [Running Multiple MPI Applications on a node](#Running-Multiple-MPI-Applications-on-a-node)
+    - [Binding MPI ranks to GPUs](#Binding-MPI-ranks-to-GPUs)
+    - [Running Multiple Multi-node Applications](#Running-Multiple-Multi-node-Applications)
+    - [Using Fakeroot with Singularity](#Using-Fakeroot-with-Singularity)
+
+
```

Comment:
We should add how to target specific nodes to this, like an example with both

`-l select=1:vnode=edtb-01[0]+1:vnode=edtb-04[1]`

and `-l select=1:vnode=edtb-01[0]:ncpus=4+3:ncpus=4`

or equivalent.  This is a really common question.

Also probably a good place to add "you cannot request to exclude a node" and to report suspect nodes.

Reply:
addressed in c9d2a7cc11f2efcece42ffbf7ed4576d15e635c5